From c43c119cc40a08b03ef18029adac6e7f44356d03 Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Wed, 24 Jul 2019 10:25:27 +0800
Subject: [PATCH] Add var var_accounts_passwords_pam_tally2_deny

Add var var_accounts_passwords_pam_tally2_deny which used by 
rule accounts_passwords_pam_tally2_deny

Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 .../var_accounts_passwords_pam_tally2_deny.var         | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)
 create mode 100644 linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/var_accounts_passwords_pam_tally2_deny.var

diff --git a/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/var_accounts_passwords_pam_tally2_deny.var b/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/var_accounts_passwords_pam_tally2_deny.var
new file mode 100644
index 0000000..e612d77
--- /dev/null
+++ b/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/var_accounts_passwords_pam_tally2_deny.var
@@ -0,0 +1,18 @@
+documentation_complete: true
+
+title: fail_deny
+
+description: 'Number of failed login attempts before account lockout with tally2'
+
+type: number
+
+operator: equals
+
+interactive: false
+
+options:
+    10: 10
+    3: 3
+    5: 5
+    6: 6
+    default: 3
-- 
1.9.1

